src/assets/images/prx-logo.svg

```diff
@@ -1,7 +1,7 @@
 <svg width="200" height="71" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="1.414">
   <path d="M181.265 37.44c0-16.185-10.37-28.342-16.953-34.433l-1.267 1.736c4.393 7.24 10.803 19.89 10.803 32.12 0 10.77-6.233 21.872-11.274 29.086l1.543 2.084c7.325-6.34 17.148-17.322 17.148-30.595zM199.953 37.44c0-16.185-10.372-28.342-16.956-34.433l-1.266 1.736c4.395 7.24 10.802 19.89 10.802 32.12 0 10.77-6.23 21.872-11.27 29.086l1.544 2.084c7.323-6.34 17.147-17.322 17.147-30.595z" fill="#e6e6e6" fill-rule="nonzero"/>
-  <path d="M14.963 47.483V6.593C5.907 13.022 0 23.582 0 35.522c0 11.94 5.907 22.502 14.963 28.932V47.483zM78.692 2.995c3.797 0 6.968.27 9.503.81C90.723 4.34 93 5.31 95.033 6.697c2.275 1.573 4.058 3.56 5.348 5.946 1.278 2.386 1.91 5.81 1.91 9.48 0 4.968-1.13 9.567-3.4 12.913-2.28 3.347-5.403 5.317-9.39 7.01l15.502 20.365 16.91-26.888s-12.638-20.777-17.064-27.028A35.423 35.423 0 0 0 81.833.046c-5.05 0-9.853 1.054-14.203 2.95h11.062z" fill="#9c9c9c" fill-rule="nonzero"/>
+  <path d="M14.963 47.483V6.593C5.907 13.022 0 23.582 0 35.522c0 11.94 5.907 22.502 14.963 28.932V47.483zM78.692 2.995c3.797 0 6.968.27 9.503.81C90.723 4.34 93 5.31 95.033 6.697c2.275 1.573 4.058 3.56 5.348 5.946 1.278 2.386 1.91 5.81 1.91 9.48 0 4.968-1.13 9.567-3.4 12.913-2.28 3.347-5.403 5.317-9.39 7.01l15.502 20.365 16.91-26.888s-12.638-20.777-17.064-27.028A35.423 35.423 0 0 0 81.833.046c-5.05 0-9.853 1.054-14.203 2.95h11.062z" fill="white" fill-rule="nonzero"/>
   <path d="M151.126 8.47l-16.68 27.054 17.015 26.754c7.48-6.505 12.205-16.08 12.205-26.76 0-10.84-4.868-20.541-12.54-27.05z" fill="#e6e6e6" fill-rule="nonzero"/>
```

Comment:
totally unqualified to decide this ... but i think we should make the whole thing white.